cTask.c

```diff
@@ -66,14 +66,87 @@ Create function for build histogram the number of symbols on the null terminated
 The symbols order must be from the most frequently symbol to the less frequently symbol
 Histogram example for sente string "A.Aa-.aA"
 
-*    
-***   
+*
+***
 ****
 Aa.-
 
 */
 void Task2_symbolFrqHistogram(uint8_t buff[])
 {
+    char TempBuffer[62]; // Array for A-Z, a-z, 0-9 and symbols ".", "!", "?", ":", "-", ",", ";", "(", ")", " "
+    char OutputBuffer[62][2];
+    uint8_t Counter_1, Counter_2 = 0;
+    uint8_t tmp, tmp1 = 0;
+    uint8_t CounterCompare = 0;
+// filling array symbols from string
+    while(buff[Counter_1] != '\0')
+    {
+        if(strchr(TempBuffer, buff[Counter_1]) == 0)
```

Comment:
Nice solution, but usually such task (for study purposes) don't use standard functions.